### DISCUSSION TITLE
How to customize the look of the sidebar?

### DISCUSSION THREAD

**Discussion by TonyHoanTrinh** [storybookjs/storybook#15639] [Help]
Hello,

I currently have a pretty standard looking sidebar:

![image](https://user-images.githubusercontent.com/26545361/126377251-dd020e59-70ce-4c76-ab78-4860c1444fb5.png)

But I wish to make it more customized in terms of it's look. Such as changing the look of the searchbar, font of the labels, having a different collapsible and etc.

I want it to look like this:

![image](https://user-images.githubusercontent.com/26545361/126377374-0da308a6-6b93-401d-bc0a-f551ab1c8282.png)

Is there anyway to customize the look of the sidebar in Storybook?
Thank you so much!

**Comment by marko-hologram** [comment]
I'm not sure that you can change so much easily, but theming docs should get you started for sure https://storybook.js.org/docs/react/configure/theming
